src/main/java/com/reckue/post/controllers/RatingController.java

```diff
@@ -0,0 +1,99 @@
+package com.reckue.post.controllers;
+
+import com.reckue.post.controllers.apis.RatingApi;
+import com.reckue.post.models.Rating;
+import com.reckue.post.services.RatingService;
+import com.reckue.post.transfers.RatingRequest;
+import com.reckue.post.transfers.RatingResponse;
+import com.reckue.post.utils.converters.RatingConverter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.reckue.post.utils.converters.RatingConverter.convert;
+
+/**
+ * Class RatingController is responsible for processing incoming requests.
+ *
+ * @author Iveri Narozashvili
+ */
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/rating")
+@CrossOrigin(origins = "*", allowedHeaders = "*")
+public class RatingController implements RatingApi {
+    private final RatingService ratingService;
+
+    /**
+     * This type of request allows to create and process it using the converter.
+     *
+     * @param ratingRequest the object of class TagRequest
```

Comment:
Please, correct javadocs. There are TagRequest and TagResponse instead of RatingRequest and RatingResponse.
As well as make reformat code.